tools/common/schema/updatetask.go

```diff
@@ -80,6 +77,7 @@ const (
 
 var (
 	whitelistedCQLPrefixes = [4]string{"CREATE", "ALTER", "INSERT", "DROP"}
+	versionDirectoryRegex = regexp.MustCompile(`^v[\d.]+`)
```

Comment:
does this also match `v.3` ?

Reply:
It does, but it would fail to parse as a semver. The only reason that we need this regex is to enforce the 'v' prefix. Otherwise we should just leave parsing up to the semver library.